docs/commandline/create.md

```diff
@@ -0,0 +1,57 @@
+# create
+
+Command `create` is used to create a container according to user's configuration.
+
+## Description
+
+The `pouch create` command creates a container object in Pouchd. When creating, all configuration user input will be stored in memory store of Pouchd. This is useful when you wish to create a container configuration ahead of time so that Pouchd will preserve the resource in advance. The container you created is ready to start when you need it.
+
+## Help Information
```

Comment:
ideally this should be generated. or it would be very hard to keep in-sync with the code.

Reply:
Yeah, I totally agree with you. @xiang90 
Then do you have any tools to generate this?
And I will spend some time in learning cobra package's auto cli doc generation at the same time.

Reply:
I tried package `github.com/spf13/cobra/doc` to generated documents here. The generated document is like the following format:

--------------------------------

## create

Create a new container with specified image

### Synopsis


Create a new container with specified image

```
create [image] [flags]
```

### Options

```
  -h, --help             help for create
      --name string      Specify name of container
  -t, --tty              Allocate a tty device
  -v, --volume strings   Bind mount volumes to container
```

###### Auto generated by spf13/cobra on 4-Dec-2017

--------------------------------

for pure command side, I think this is good. But for all cli side command usage, this is not sufficient. We need to add more description, illustration and examples. WDYT? @xiang90